lib/pluginmanager/prepare_offline_pack.rb

```diff
@@ -21,7 +22,21 @@ def execute
     # To silence some of debugs/info statements
     Paquet.ui = Paquet::SilentUI unless debug?
 
-    FileUtils.rm_rf(output) if ::File.exist?(output)
+    if File.directory?(output)
+      signal_error("The specified output is a directory, you must specify a filename with a zip extension, provided output: #{output}")
+    else
+      if File.extname(output) !~ /zip/i
+        signal_error("You must specify the zip extension for the provided filename: #{output}")
+      end
+
+      if ::File.exists?(output)
+        if overwrite?
+          File.delete(output)
+        else
+          signal_error("Package creation cancelled, a previously generated package exist at location: #{output}, move this file to safe place and run the command again")
```

Comment:
i suggest making this three messages more consistent. only this one starts saying `Package creation cancelled`

We can also just say "Package creation cancelled: output file destination #{} already exists".